### PR TITLE
Add details about VMs using virsh

### PR DESCRIPTION
**Pull request by assafad**
Extend kubevirt must-gather collection to collect more info with virsh command.

Which issue(s) this PR fixes:
https://issues.redhat.com/browse/CNV-26740

**Release note**:

```release-note
Added capabilities.xml
Added domcapabilities.xml
Added list.txt 
Added domblklist.txt 
Added domjobinfo.txt
Added blockjob.txt
```

### CODE REVIEW

File: collection-scripts/gather_vms_details [hunk 1]
```diff
@@ -19,8 +19,29 @@ function gather_vm_info() {
...
+  /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh -r domblklist "${ocproject}_${vmname}" > "${vm_collection_path}/${ocvm}.domblklist.txt"
+
+  # VM : domjobinfo
+  /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh -r domjobinfo "${ocproject}_${vmname}" > "${vm_collection_path}/${ocvm}.domjobinfo.txt"
+
+   # VM : blockjob
+#   for disk in $(cat "${vm_collection_path}/${ocvm}.domblklist.txt" | awk '{print $2}'|grep -v Source):
+#    do
+#      /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh -r blockjob "${ocproject}_${vmname}" $disk > "${vm_collection_path}/${ocvm}.blockjob.txt"
+#    done
```
Comment (assafad) [inline collection-scripts/gather_vms_details:44]: @tiraboschi Hi, this part creates an empty `.blockjob.test` file, although the loop seems to be working. Any idea why?

**Comment by assafad** [comment]
@enp0s3 Hi, can you please take a look?

**Comment by assafad** [comment]
Hi @nunnatsa/ @tiraboschi, can you please take a look?

File: collection-scripts/gather_vms_details [hunk 2]
```diff
@@ -19,8 +19,28 @@ function gather_vm_info() {
...
+  /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh -r dumpxml "${ocproject}_${vmname}" > "${vm_collection_path}/${ocvm}.dumpxml.xml"
+
+  # VM : domblklist
+  /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh -r domblklist "${ocproject}_${vmname}" > "${vm_collection_path}/${ocvm}.domblklist.txt"
+
+  # VM : domjobinfo
+  /usr/bin/oc exec "${ocvm}" -n "${ocproject}" -c compute -- virsh -r domjobinfo "${ocproject}_${vmname}" > "${vm_collection_path}/${ocvm}.domjobinfo.txt"
+
+  # VM : blockjob
+  for disk in $(cat "${vm_collection_path}/${ocvm}.domblklist.txt" | awk '{print $2}'|grep -v Source); do
```
Comment (tiraboschi) [inline collection-scripts/gather_vms_details:41]: can you please address shellcheck violations?

**Review by tiraboschi — APPROVED** [review]

**Pull request merged**